dist/index.js

```diff
@@ -1907,6 +1907,14 @@ const runTests = () => {
       cmd.push(browser)
     }
 
+    const envInput = core.getInput('env')
+    if (envInput) {
+      // TODO should env be quoted?
+      // If it is a JSON, it might have spaces
+      cmd.push('--env')
+      cmd.push(envInput)
```

Comment:
hmm, yes, might need quotes, maybe single quotes, ughh, will require testing across OS

Reply:
When entering JSON it should already be wrapped in it's own string right?

For now do you think we should add a note to the README about potential issues or did you want to improve the handling/testing before releasing it?